lbz/authz.py

```diff
@@ -63,7 +65,16 @@ def set_resource(self, resource_name):
 
     def validate(self, function_name):
         if function_name not in self._permissions:
-            raise ServerError
+            raise NotAcceptable()
+        if self.expiration is None:
+            warnings.warn(
+                "EXPIRATION_KEY will be mandatory with 0.2 please upgrade Authz provider",
+                DeprecationWarning,
+            )
+        elif self.expiration < datetime.utcnow().timestamp():
+            raise PermissionDenied(
+                f"Your token has expired at {self.expiration} . Please refresh it."
```

Comment:
Would be nice to convert the timestamp to actual datetime string before displaying, also there is an unnecessary space before the first period